src/app/shell/details/achievements/achievements.component.html

```diff
@@ -1,6 +1,16 @@
+<div *ngIf="auth" class="add-wrapper">
```

Comment:
auth &&  workshop.providerId= provider.id
(you can add a new achievements to the workshop only this workshop belongs to your provider)